src/Akihabara/Native/Framework/UnsafeCalculatorGraph.cs

```diff
@@ -32,7 +34,7 @@ public static extern MpReturnCode
         [DllImport(MediaPipeLibrary, ExactSpelling = true)]
         public static extern MpReturnCode mp_CalculatorGraph__ObserveOutputStream__PKc_PF(IntPtr graph,
             string streamName,
-            [MarshalAs(UnmanagedType.FunctionPtr)] CalculatorGraph.NativePacketCallback packetCallback,
+            [MarshalAs(UnmanagedType.FunctionPtr)] NativePacketCallback packetCallback,
```

Comment:
How was this sourced? if you referenced on the Unity repo, there's a calculator graph class generated from a protofile. We shouldn't touch it unless you know what it is.

Reply:
Should we just integrate this class from the Unity repo into `ProtoCalculator`?

Reply:
Did you check `build.py`? It's getting some protofiles and protocs them, so figure that one out first.

Reply:
This was sourced here, 
https://github.com/homuler/MediaPipeUnityPlugin/blob/92dd55579f2faeabf49338974ddb1d15ae54b683/Packages/com.github.homuler.mediapipe/Runtime/Scripts/Framework/CalculatorGraph.cs#L8

MediaPipeUnity's "`CalculatorGraph.cs`" is just a shim for `CalculatorGraph_Unsafe.cs`

Reply:
What's `CalculatorGraph_Unsafe.cs`?

Reply:
We're looking at this a bit incorrectly, you can refer here: https://github.com/homuler/MediaPipeUnityPlugin/issues/235

Reply:
Does this mean that we already have what it takes? It seems like it's defined in the C#-generated protobuf code from the previous pull request:
![image](https://user-images.githubusercontent.com/13885008/133108372-cf5e7be0-4d73-4750-b6aa-972919da17f6.png)

Reply:
Looks like it. Is this ready to be merged if that's the only concern?

Reply:
Well, if we undo the modification above, Akihabara simply won't compile. From where then are we supposed to take the `NativePacketCallback` type?

Reply:
> Well, if we undo the modification above, Akihabara simply won't compile. From where then are we supposed to take the `NativePacketCallback` type?

Check from the C code to see if that's correct type for the delegate, if it is, then I can approve it.

Reply:
This is the code from `mediapipe_api/framework/calculator_graph.cc` (l.69):
```cpp
MP_CAPI(MpReturnCode) mp_CalculatorGraph__ObserveOutputStream__PKc_PF(mediapipe::CalculatorGraph* graph,
                                                                      const char* stream_name,
                                                                      NativePacketCallback* packet_callback,
                                                                      absl::Status** status_out) {
  TRY {
    auto status = graph->ObserveOutputStream(stream_name, [packet_callback](const mediapipe::Packet& packet) -> ::absl::Status {
      return absl::Status { std::move(*packet_callback(packet)) };
    });
    *status_out = new absl::Status { std::move(status) };
    RETURN_CODE(MpReturnCode::Success);
  } CATCH_EXCEPTION
}
```

The `NativePacketCallback` type is defined under `mediapipe_api/framework/calculator_graph.h` (l.23) as the following function:
```cpp
typedef absl::Status* NativePacketCallback(const mediapipe::Packet&);
```

Reply:
That returns a Integer so `IntPtr` is correct. Thanks for investigating.